test/runner/lib/executor.py

```diff
@@ -796,6 +796,19 @@ def command_sanity_validate_modules(args, targets):
     if skip_paths:
         cmd += ['--exclude', '^(%s)' % '|'.join(skip_paths)]
 
+    # Find fork_branch
+    git = Git(args)
+    try:
+        changes = ShippableChanges(args, git) if is_shippable() else LocalChanges(args, git)
+    except ApplicationError:
```

Comment:
I'll see what I can do.  I'm actually fine with ignoring local largely because I doubt many are running the sanity checks locally anyway, so this should be fine, to only catch in shippable.

I'll update with your suggestion.